csfieldguide/interactives/content/en/interactives.yaml

```diff
@@ -144,6 +144,8 @@ searching-algorithms:
   name: Searching Algorithms
 sha2:
   name: SHA2
+short-term-memory-picture:
+ name: Short Term Memory Picture Game
```

Comment:
Don't call it either of these things; it looks out of place on the interactives page. Try 'Memory Pictures' or something